spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -175,6 +175,123 @@ def refresh_views
     end
   end
 
+  context "show v2" do
+    render_views_on_ci
```

Comment:
rendering views in a local TDD loop was too painful, and I'm not really testing anything here deep in the views anyways other than the fact that they render and don't blow up